sentinel-core/src/main/java/com/alibaba/csp/sentinel/util/VersionUtil.java

```diff
@@ -0,0 +1,21 @@
+package com.alibaba.csp.sentinel.util;
+
+import com.alibaba.csp.sentinel.log.RecordLog;
+
+/**
+ * Get Sentinel version from MANIFEST.MF
+ * 
+ * @author jason<jason@dayima.com> @ Sep 28, 2018
```

Comment:
You can remove the redundant author information (email, date) and only leave name here. Also, add a `@since 0.2.1` here to indicate this is a new class since next version. Please also modify this in other new-created classes.